internal/infrastructure/db/migrations/20221020065955_integration_features_weight.sql

```diff
@@ -0,0 +1,36 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+ALTER TABLE integration_features ADD feature_weight float NULL;
```

Comment:
nitpick: If we write `DROP COLUMN`, why not `ADD COLUMN`?

Reply:
its not really a requirement for postgres

Reply:
It's not required, I just like regularity.